docs/mkcloud.md

```diff
@@ -71,8 +71,80 @@ To use mkcloud the following additional steps are needed:
 
 ## mkcloud
 
-`mkcloud` can be configured using different environment variables.  Some of
-them are listed below.  To get a complete list run: `./mkcloud -h`
+`mkcloud` consists of different steps which can be run independently, provided the order is logical.
+```
+   all -> expands to: cleanup prepare setupadmin addupdaterepo runupdate prepareinstcrowbar instcrowbar rebootcrowbar setupcompute instcompute proposal testsetup rebootcompute
+   all_noreboot -> exp to: cleanup prepare setupadmin addupdaterepo runupdate prepareinstcrowbar instcrowbar setupcompute instcompute proposal testsetup
+   plain    -> expands to: cleanup prepare setupadmin prepareinstcrowbar instcrowbar setupcompute instcompute proposal
+   plain_with_upgrade  -> expands to: cleanup prepare setupadmin prepareinstcrowbar instcrowbar setupcompute instcompute proposal prepare_cloudupgrade cloudupgrade_1st cloudupgrade_2nd cloudupgrade_clients cloudupgrade_reboot_and_redeploy_clients
+   instonly -> expands to: cleanup prepare setupadmin prepareinstcrowbar instcrowbar setupcompute instcompute
+
+   cleanup:       kill all running VMs, zero out boot sectors of all LVM volumes
+   prepare:       create LVM volumes, setup libvirt networks
+   setupadmin:    create the admin node and install the cloud product
+   prepareinstcrowbar: add repos and install crowbar packages
+   instcrowbar:   install crowbar and chef on the admin node
+   setupcompute:  create the compute nodes and let crowbar discover them
+   instcompute:   allocate and install compute nodes
+   proposal:      create and apply proposals for default setup
+   setuplonelynodes: boot a number (defined by nodenumberlonelynodes) of non-crowbar registered nodes in the admin network
+   crowbar_register: register a number (defined by nodenumberlonelynodes) of non-crowbar nodes with crowbar (setuplonelynodes needs to have run before)
+   testsetup:     start a VM in the cloud
+   addupdaterepo: addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
+   runupdate:     run zypper up on the crowbar node
+                  (compute nodes are automaticallyupdated via chef run)
+   rebootcrowbar: reboot the crowbar instance and wait for it being up
+   rebootcompute: reboot the compute nodes and wait for them being up
+   restartcloud:  start a pre-existing cloud again after host reboot
+   securitytests: install and run security test suite
+   qa_test:       run the qa test suite
+   help:          this usage
+```
+
+`mkcloud` can be configured using different environment variables.
+To get a complete list run: `./mkcloud -h`
```

Comment:
Again I'm wary of duplicating documentation in two places.  I think we should decide on one location and stick to ensuring the docs are correct in that place.